.github/workflows/int-test-workflow.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - master
+      - cspl-1851
```

Comment:
just putting out a comment here so that we don't forget to remove this once all tests pass!